diff --git a/wled00/const.h b/wled00/const.h
index 38861af..7e048ad 100644
--- a/wled00/const.h
+++ b/wled00/const.h
@@ -254,6 +254,7 @@ static_assert(WLED_MAX_BUSSES <= 32, "WLED_MAX_BUSSES exceeds hard limit");
 #define USERMOD_ID_RF433                 56     //Usermod "usermod_v2_RF433.h"
 #define USERMOD_ID_BRIGHTNESS_FOLLOW_SUN 57     //Usermod "usermod_v2_brightness_follow_sun.h"
 #define USERMOD_ID_USER_FX               58     //Usermod "user_fx"
+#define USERMOD_ID_DUEPOINT_ACTUATOR      59     //Out-of-tree DuePoint ventilation actuator
 
 //Wifi encryption type
 #ifdef WLED_ENABLE_WPA_ENTERPRISE
diff --git a/wled00/pin_manager.h b/wled00/pin_manager.h
index 7bdd5cf..a49b02c 100644
--- a/wled00/pin_manager.h
+++ b/wled00/pin_manager.h
@@ -76,7 +76,8 @@ enum struct PinOwner : uint8_t {
   UM_LDR_DUSK_DAWN     = USERMOD_ID_LDR_DUSK_DAWN,      // 0x2B // Usermod "usermod_LDR_Dusk_Dawn_v2.h"
   UM_MAX17048          = USERMOD_ID_MAX17048,           // 0x2F // Usermod "usermod_max17048.h"
   UM_BME68X            = USERMOD_ID_BME68X,             // 0x31 // Usermod "usermod_bme68x.h -- Uses "standard" HW_I2C pins
-  UM_PIXELS_DICE_TRAY  = USERMOD_ID_PIXELS_DICE_TRAY    // 0x35 // Usermod "pixels_dice_tray.h" -- Needs compile time specified 6 pins for display including SPI.
+  UM_PIXELS_DICE_TRAY  = USERMOD_ID_PIXELS_DICE_TRAY,   // 0x35 // Usermod "pixels_dice_tray.h" -- Needs compile time specified 6 pins for display including SPI.
+  UM_DuePointActuator  = USERMOD_ID_DUEPOINT_ACTUATOR   // 0x3B // Out-of-tree DuePoint actuator
 };
 static_assert(0u == static_cast<uint8_t>(PinOwner::None), "PinOwner::None must be zero, so default array initialization works as expected");
 
